### PR TITLE
Fix integrators name when setting the Clusters updater.

### PR DESCRIPTION
**Pull request by lyrivera**
## Description
Although the Clusters updater has already been ported to v3.0 API, it still had some legacy integrator names.

## Motivation and context

Resolves: None

## How has this been tested?
No tests required, only renaming involved.

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/master/ContributorAgreement.md).
- [x] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

**Review by joaander — COMMENTED** [review]
Thanks! Please add a parameterized unit test that verifies that these schedule #702.

**Comment by lyrivera** [comment]
> Thanks! Please add a parameterized unit test that verifies that these schedule #702.

Will do.

**Comment by lyrivera** [comment]
Update: I found a few other issues with the new python API that I will address in this PR. I will also include unit tests here.

**Comment by lyrivera** [comment]
All the tests are passing except for one that is crashing with the following error (it only fails in MPI):

```
hoomd/hpmc/pytest/test_clusters.py:164: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
hoomd/conftest.py:47: in make_simulation
    sim.create_state_from_snapshot(snapshot)
hoomd/simulation.py:140: in create_state_from_snapshot
    self._state = State(self, snapshot)
hoomd/state.py:64: in __init__
    snapshot._broadcast_box()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <hoomd.snapshot.Snapshot object at 0x7fe979dd4d90>

    def _broadcast_box(self):
>       self._cpp_obj._broadcast_box(self._comm.cpp_mpi_conf)
E       RuntimeError: Failed to read 8 bytes from input stream! Read 4

hoomd/snapshot.py:114: RuntimeError
```

I haven't been able to figure out what is causing this error but I'm able to reproduce it locally in my laptop when executing `pytest` with MPI. However, if I export the test to a standalone script and execute it with MPI, it runs without errors.

**Comment by lyrivera** [comment]
@joaander I ran the tests locally with `mpirun -n 2 hoomd/pytest/pytest-openmpi.sh -v -x hoomd` as suggested in our private conversation. However, I'm still getting the same error in the `test_swap_moves` test. Again, it only fails when running in multiple ranks but passes without problems in a single rank and non-mpi executions.

**Comment by joaander** [comment]
What is the output in the pytest.out.1 file?

**Comment by joaander** [comment]
@jglaser Are we keeping swap moves in your rewritten clusters updater?

**Comment by lyrivera** [comment]
> What is the output in the pytest.out.1 file?

hoomd/hpmc/pytest/test_clusters.py::test_swap_moves[CPU--2.0] PASSED     [ 76%]
hoomd/hpmc/pytest/test_clusters.py::test_swap_moves[CPU-0] FAILED        [ 76%]

=================================== FAILURES ===================================
____________________________ test_swap_moves[CPU-0] ____________________________

delta_mu = 0
simulation_factory = <function simulation_factory.<locals>.make_simulation at 0x10d48f160>
lattice_snapshot_factory = <function lattice_snapshot_factory.<locals>.make_snapshot at 0x10de1db80>

    @pytest.mark.parametrize("delta_mu", [-2.0, 0, 2.0])
    def test_swap_moves(delta_mu, simulation_factory,
                        lattice_snapshot_factory):
        """Test that Clusters can swap types."""

>       sim = simulation_factory(lattice_snapshot_factory(particle_types=['A', 'B'],
                                                          dimensions=3, a=4, n=7, r=0.1))

hoomd/hpmc/pytest/test_clusters.py:164:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
hoomd/conftest.py:47: in make_simulation
    sim.create_state_from_snapshot(snapshot)
hoomd/simulation.py:140: in create_state_from_snapshot
    self._state = State(self, snapshot)
hoomd/state.py:64: in __init__
    snapshot._broadcast_box()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <hoomd.snapshot.Snapshot object at 0x10dff7130>

    def _broadcast_box(self):
>       self._cpp_obj._broadcast_box(self._comm.cpp_mpi_conf)
E       RuntimeError: Failed to read 8 bytes from input stream! Read 4

hoomd/snapshot.py:114: RuntimeError
=========================== short test summary info ============================
FAILED hoomd/hpmc/pytest/test_clusters.py::test_swap_moves[CPU-0] - RuntimeEr...
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
======================= 1 failed, 1097 passed in 13.70s ========================

File: hoomd/hpmc/pytest/test_clusters.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    sim.state.snapshot = snap
+
+    # number of particles of each type should change after a run
+    sim.run(100)
+
+    snap = sim.state.snapshot
+    if snap.exists:
+        num_type_B = np.sum(snap.particles.typeid)
+        num_type_A = len(snap.particles.typeid) - num_type_B
+        acceptance = cl.swap_moves[0]/np.sum(cl.swap_moves)
```
Comment (joaander) [inline hoomd/hpmc/pytest/test_clusters.py:193]: Accessing properties like `swap_moves` must be done outside the `if snap.exists`. These are collective operations that must be performed on all ranks. What is happening here is that a message from `swap_moves` on one rank is getting received by `broadcast_box` on the other rank which skipped past this conditional.

Comment (lyrivera) [reply hoomd/hpmc/pytest/test_clusters.py]: Whoops, I totally missed that. Seems to be working now. Thanks!

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**